spec/marc_to_solr/lib/princeton_marc_spec.rb

```diff
@@ -664,6 +668,14 @@ def fixture_record(fixture_name)
       @holdings_scsb_block = @holdings_scsb[@holding_id_scsb]
     end
 
+    it 'indexes location if it exists' do
+      expect(@oversize_holding_block['location']).to eq 'Stacks'
+    end
+
+    it 'if location is blank it will index blank' do
+      expect(@holdings_no_location_name_holding_block["location"]).to eq ''
+    end
```

Comment:
This is a case mms_id : 99116547863506421, where the location code `eastasian$hy` has blank label (blank Alma external name) in bibdata. The `location` we index in holdings1_display is blank.